actors/evm/src/lib.rs

```diff
@@ -51,6 +51,7 @@ fn test_method_selector() {
 #[repr(u64)]
 pub enum Method {
     Constructor = METHOD_CONSTRUCTOR,
+    // TODO: Do we want to use ExportedNums for all of these, per FRC-42?
```

Comment:
For `InvokeContract` and `GetBytecode`, probably. But we still need to figure that out.

Reply:
https://github.com/filecoin-project/builtin-actors/issues/968